server/controller/controller.js

```diff
@@ -0,0 +1,96 @@
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt');
+
+const User = require('../config/sequelize');
+const hashPassword = require('../config/hashPassword');
+const totp = require('../config/totp');
+
+exports.signin =  (req, res) => {
+
+  User.findOne({
+    where : {
+      email : req.body.email
+    }
+  }).then(user => {
+    if(!user) {
+      return res.status(404).json({msg :'user is not found'});
+    }
+
+    const passwordIsValid = bcrypt.compareSync(req.body.password, user.password);
+    
+    if(!passwordIsValid) {
+      return res.status(401).json({auth: false, token : 'invalid', msg : 'password is incorrect'});
+    }
+
+    const token = jwt.sign({id : user.id}, process.env.SECRET, {
+      expiresIn: 86400
+    });
+
+    res.status(200).json({auth : true, token});
+  }).catch(err => {
+    res.status(500).json({msg : "invalid email"})
+  })
+}
```

Comment:
Make the error messages proper eg: "invalid email" - > "Invalid e-mail". i.e make the sentences more reader friendly and understandable, make them more descriptive if you can.

Reply:
I will work on this.